CHANGELOG.md

```diff
@@ -1,26 +1,24 @@
 # Changelog
 
-All notable changes to Kubernetes and Helm resources for WSO2 API Management version `1.5.x` in each resource release,
+All notable changes to Kubernetes and Helm resources for WSO2 Open Banking version `2.0.0` in each resource release,
 will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [v1.5.0.1] - 2020-09-25
+## [v2.0.0.1] - 2021-01-22
```

Comment:
@nalakagunathilaka let's set the date just prior to sending the `master` merge PR.

Reply:
Will set a final commit to set the date when all comments are resolved and ready merge to send the merge PR to master as well.

Reply:
+1